feature-libs/product-configurator/rulebased/components/config/configurator-ui-settings.ts

```diff
@@ -0,0 +1,13 @@
+export abstract class ConfiguratorUISettings {
+  rulebasedConfigurator: {
+    quantityDebounceTime?: number;
+    inputDebounceTime?: number;
+  };
+}
+
+export const DefaultConfiguratorUISettings: ConfiguratorUISettings = {
```

Comment:
[nitpick] for consistency let's use camelCase for consts and functions. Pascal case for classes

Reply:
pick